src/server/graphql/types/ThinkPhase.js

```diff
@@ -0,0 +1,29 @@
+import {GraphQLID, GraphQLList, GraphQLNonNull, GraphQLObjectType} from 'graphql';
+import NewMeetingPhase, {newMeetingPhaseFields} from 'server/graphql/types/NewMeetingPhase';
+import RetroPhaseItem from 'server/graphql/types/RetroPhaseItem';
+import GenericMeetingStage from 'server/graphql/types/GenericMeetingStage';
+
+const ThinkPhase = new GraphQLObjectType({
+  name: 'ThinkPhase',
+  description: 'The meeting phase where all team members check in one-by-one',
+  interfaces: () => [NewMeetingPhase],
+  fields: () => ({
+    ...newMeetingPhaseFields(),
+    focusedPhaseItemId: {
```

Comment:
Following the Meeting -> Phase -> Stage hierarchy, we can store state wherever it makes sense.
for example, where is the facilitator? That's meeting state
which column is highlighted during the think phase? that state only belongs to a single phase
did we already discuss the 3rd card grouping? that state belongs on the stage (isComplete = true)